src/app/(pages)/nominations/[walletId]/page.tsx

```diff
@@ -0,0 +1,34 @@
+import { Metadata } from "next";
+import { fetchMetadata } from "frames.js/next";
+import { FRAMES_BASE_PATH, appURL } from "@/shared/frames/utils";
+import Nothing from "../../@modal/(home)/page";
+
+const description = [
+  "BUILD is a meme and a social game designed to reward builders via",
+  "onchain nominations.",
+].join(" ");
+
+export async function generateMetadata({
+  params: { walletId },
+}: {
+  params: { walletId: string };
+}): Promise<Metadata> {
+  return {
+    title: "BUILD",
+    description: description,
+    openGraph: {
+      title: "BUILD",
+      description: description,
+      type: "website",
+      url: "https://build.top",
+      images: ["https://build.top/images/BUILD-thumbnail.jpg"],
+    },
+    other: {
+      ...(await fetchMetadata(
+        new URL(FRAMES_BASE_PATH + `/nominations/${walletId}`, appURL()),
+      )),
+    },
+  };
+}
+
+export default Nothing;
```

Comment:
It's better to have this page with a client side redirect. something like this will do: 

```tsx
"use client"; 

export default RedirectToRoot() {
  useEffect(() => {
    router.replace("/");
  })
  
  return null;
}
```